docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -38,7 +38,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        # this will assign the start and stop time of the first epoch in the 'stim_table' to the start and stop variables
</pre><p>can be less verbose. comments are best when they can be brief. Instead of adding to the original comment that says "specify a current_epoch...", you can replace it</p><p> </p><p>When writing comments to describe lines of code, its good to explain in abstract <em>why</em> the line is there rather than <em>what</em> it is doing, or at least a description which might hint at the why. Normally programmers can figure out what it is doing, the why is usually the elusive part.</p><p> </p><p>you could say something like</p><p>"set first epoch bounds to first start and stop times in stim table"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>